src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import { ReactElement, PropsWithChildren } from 'react';
+
+export interface InputDropdownProps extends PropsWithChildren {
```

Comment:
please check the name of the interface and the component name

Reply:
Renamed. All are titled with the same name.